src/clique_command.erl

```diff
@@ -40,23 +40,23 @@ init() ->
     ok.
 
 %% @doc Register a cli command (i.e.: "riak-admin handoff status")
--spec register([string()], list(), list(), fun()) -> true.
+-spec register(['*' | string()], list(), list(), fun()) -> true.
```

Comment:
Thanks for the clarification Nick.

On Mon, Jun 22, 2015 at 7:10 AM, Nick Marino <notifications@github.com>
wrote:

> In src/clique_command.erl
> <https://github.com/basho/clique/pull/58#discussion_r32937344>:
>
> > @@ -40,23 +40,23 @@ init() ->
> >      ok.
> >
> >  %% @doc Register a cli command (i.e.: "riak-admin handoff status")
> > --spec register([string()], list(), list(), fun()) -> true.
> > +-spec register(['*' | string()], list(), list(), fun()) -> true.
>
> Thanks, that's a great idea. I'll add a check for that shortly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/basho/clique/pull/58/files#r32937344>.
>



-- 
Heather McKelvey
VP Engineering
Basho Technologies